Makefile

```diff
@@ -1,39 +1,24 @@
-TGIS_IMAGE=quay.io/opendatahub/text-generation-inference:stable-bafd218
-
-.PHONY: default
+.PHONY: default refresh-poetry-lock-files
 
 default:
-	podman build -t caikit-tgis-serving:$$(git rev-parse --short HEAD) .
-
+	podman build \
+		-t opendatahub/caikit:latest \
```

Comment:
On an M1 Mac and podman this fails with this error:

```text
copying psutil/tests/test_windows.py -> build/lib.linux-aarch64-cpython-39/psutil/tests
  running build_ext
  building 'psutil._psutil_linux' extension
  creating build/temp.linux-aarch64-cpython-39
  creating build/temp.linux-aarch64-cpython-39/psutil
  gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=595 -DPy_LIMITED_API=0x03060000 -DPSUTIL_ETHTOOL_MISSING_TYPES=1 -DPSUTIL_LINUX=1 -I/tmp/tmpcp388sgs/.venv/include -I/usr/include/python3.9 -c psutil/_psutil_common.c -o build/temp.linux-aarch64-cpython-39/psutil/_psutil_common.o
  psutil could not be installed from sources because gcc is not installed. Try running:
    sudo yum install gcc python3-devel
  error: command 'gcc' failed: No such file or directory
  

  at /usr/local/lib/python3.9/site-packages/poetry/installation/chef.py:147 in _prepare
      143│ 
      144│                 error = ChefBuildError("\n\n".join(message_parts))
      145│ 
      146│             if error is not None:
    → 147│                 raise error from None
      148│ 
      149│             return path
      150│ 
      151│     def _prepare_sdist(self, archive: Path, destination: Path | None = None) -> Path:

Note: This error originates from the build backend, and is likely not a problem with poetry but with psutil (5.9.5) not supporting PEP 517 builds. You can verify this by running 'pip wheel --use-pep517 "psutil (==5.9.5)"'.

Error: building at STEP "RUN pip3 install poetry && poetry install": while running runtime: exit status 1

```

Reply:
Yeah, building in the image for multiarch will have to come separately